test/src/test/java/hudson/model/ItemGroupMixInTest.java

```diff
@@ -207,4 +207,28 @@ public boolean isApplicable(Class jobType) {
         assertThat(Items.getConfigFile(foo).asString(), containsString("<description/>"));
     }
 
+    @Test
+    public void createProjectFromXMLWithSpaces() throws IOException {
+        final String xml = "<?xml version='1.0' encoding='UTF-8'?>\n" +
```

Comment:
this test is testing much more than the spaces.  The ENTITY use should be stripped here.